packages/js/src/utils/interfaces.ts

```diff
@@ -22,8 +23,35 @@ export interface IClientOptions {
   ringFile?: string;
 }
 
-// TODO Consolidate with `CallOptions`
-export interface ICallOptions {
+/**
+ * @interface IClientOptions
+ */
+export interface IClientOptions {
+  /**
+   * The JSON Web Token (JWT) to authenticate with your SIP Connection.
+   * This is the recommended authentication strategy. [See how to create one](https://developers.telnyx.com/docs/v2/webrtc/quickstart).
+   */
+  login_token: string;
+  /**
+   * The `username` to authenticate with your SIP Connection.
+   * `login` and `password` will take precedence over
+   * `login_token` for authentication.
+   */
+  login?: string;
+  /**
+   * The `password` to authenticate with your SIP Connection.
+   */
+  password?: string;
+  /* A URL to a wav/mp3 ringtone file. */
```

Comment:
```js
/**
* A URL to a wav/mp3 ringtone file.
*/
```
it is not showing on MD file

Reply:
![image](https://user-images.githubusercontent.com/16343871/104731104-fc158600-5719-11eb-8538-951832159057.png)